tests/imports_test.go

```diff
@@ -358,6 +361,23 @@ func (t *ProviderSuite) Test_NewStreamImportSkToken() {
 	t.NoError(si.SetToken(token))
 }
 
+func (t *ProviderSuite) Test_NewStreamImportAllowTracing() {
```

Comment:
nit: No reason to have `New` in the test name? (my guess is copy/paste from previous test that had it).

Reply:
fixed